### PR TITLE
Upgrade to Laravel 10 & php 8.2 & fix Dependabot alerts

### PR DESCRIPTION
**Pull request by DanielGSoftware**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,13 +16,13 @@
     },
     "require": {
         "php": "^8.1",
-        "laravel/framework": "^v9.0",
-        "lenderspender/laravel-enums": "^1.1",
+        "laravel/framework": "^v10.0",
```
Comment (TijmenWierenga) [inline composer.json]: The `v` prefix shouldn't be here, right?

**Review by TijmenWierenga — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (DanielGSoftware) [reply composer.json]: Can you tell me why you think this?

Comment (TijmenWierenga) [reply composer.json]: There's never a `v` in composer dependencies.

https://getcomposer.org/doc/articles/versions.md#:~:text=Note%20that%20Composer%20automatically%20removes%20the%20%27v%27%20prefix%20in%20the%20actual%20tagname%20to%20get%20a%20valid%20final%20version%20number.

**Review by TijmenWierenga — APPROVED** [review]

**Review by thomasmoors — APPROVED** [review]
🎖️

**Pull request merged**